client/src/GlobalStyles.js

```diff
@@ -16,7 +16,11 @@ const GlobalStyles = () => {
         }
         :root {
           --clr-primary: #ffa200;
-          --highlight-clr-category: #fa20d8f2;
+          --highlight-clr-category1: #fa20d8;
+          --highlight-clr-category2: #0bff69;
+          --highlight-clr-category3: #fbff14;
+          --highlight-clr-category4: #ffa200;
+          --highlight-clr-category5: #33b7f2;
```

Comment:
Is this really part of this PR or could this be handled in a global PR?

Reply:
you are totally right - should have been in the eventcard branch